data_models/okta_data_model.yml

```diff
@@ -7,7 +7,7 @@ Filename: okta_data_model.py
 Enabled: true
 Mappings:
   - Name: actor_user
-    Path: $.actor.displayName
+    Path: get_actor_user
```

Comment:
Need to change `Path` to `Method` here so it uses the method you wrote

Reply:
Ah - thanks for catching!